src/test/resources/io/jenkins/plugins/opentelemetry/jcasc/configuration-as-code-expected.yml

```diff
@@ -7,4 +7,6 @@ observabilityBackends:
 - customObservabilityBackend:
     metricsVisualisationUrlTemplate: "foo"
     traceVisualisationUrlTemplate: "http://example.com"
+- zipkin:
+    zipkinBaseUrl: "http://localhost:9411/"
```

Comment:
There is a manipulation with https://github.com/jenkinsci/opentelemetry-plugin/blob/2339cbbd1ac1456b430bb2edbd9ae6a5a9a2b203/src/main/java/io/jenkins/plugins/opentelemetry/backend/ZipkinBackend.java#L47-L50